src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -87,9 +89,13 @@ public static byte[] CompileAssembly(GeneratedSyntax generatedSyntax, string ass
                     .AddSyntaxTrees(code.SyntaxTree)
                     .AddReferences(assemblies)
                     .WithOptions(options);
+            MemoryStream symbolStream = null;
             using (var stream = new MemoryStream())
+#if DEBUG
+            using (symbolStream = new MemoryStream())
+#endif
```

Comment:
Thanks @jdom, I've updated the PR to expose RuntimeVersion.IsDebugBuild & am using that instead